yaml/jobs/collections/php-rh.yaml

```diff
@@ -44,6 +44,13 @@
             context: 'rhel7 - openshift'
             not_automatic: true
             trigger_phrase: 'test-openshift'
+        - '{job_prefix}-{name}':
+            job_prefix: 'rhscl-images'
+            name: 'php-rh-openshift-4'
```

Comment:
is it expected to have it set up for php only?

Reply:
I would like to test it only for one image. The rest will follow as soon as the tests are starting well.